CHANGELOG.md

```diff
@@ -10,6 +14,30 @@
 * Support named backlinks on realm models. You can now add and annotate a realm object iterator field with `@Backlink(#fieldName)`. ([#996](https://github.com/realm/realm-dart/pull/996))
 * Allow `@Indexed` attribute on all indexable type, and ensure appropriate indexes are created in the realm. ([#797](https://github.com/realm/realm-dart/issues/797))
 * Add `parent` getter on embedded objects. ([#979](https://github.com/realm/realm-dart/pull/979))
+* Support [Client Resets](https://www.mongodb.com/docs/atlas/app-services/sync/error-handling/client-resets/). Flutter SDK automatically detects the need for client resets and automatically performs it according to the configured callbacks for the type of client reset handlers set to `FlexibleSyncConfiguration`. 
+  Added a parameter `clientResetHandler` to `Configuration.flexibleSync`. 
```

Comment:
```suggestion
* Added a parameter `clientResetHandler` to `Configuration.flexibleSync`. 
```

Reply:
This isn't a new functionality. The text just continue explaining what is changed in relation to Client reset.

Reply:
yeah that is part of the same `Support Client Reset` bullet above.

Reply:
Fixed.